src/components/01-branding/02-typography/heading/heading-default.config.yml

```diff
@@ -0,0 +1,18 @@
+label: Heading
+status: wip
+preview: "@sam"
+variants:
+  - name: default
```

Comment:
Please remove default variant. It will automatically be provided as the standard variant